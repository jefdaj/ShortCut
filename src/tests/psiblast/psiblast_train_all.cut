pcc6803 = load_faa "data/6803genes.faa"
pcc7942 = load_faa "data/7942genes.faa"
cyanodb = makeblastdb_prot [pcc6803, pcc7942]
query = load_faa "data/7942gene.faa"
pssm = psiblast_train_all 1.0e-10 query [pcc6803, pcc7942]
result = pssm
