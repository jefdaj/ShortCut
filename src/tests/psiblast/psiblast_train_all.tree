.
|-- cache
|   |-- lines
|   |   |-- 0cf27272d3.txt
|   |   |-- 3f62a639a8.txt
|   |   |-- 4d395ef46f.txt
|   |   |-- 8abd22b6c2.txt
|   |   |-- adfdb6bd8c.txt
|   |   |-- bdc783547b.txt
|   |   `-- c3ae472116.txt
|   |-- load
|   |   |-- 82645ed6f6.faa -> $WORKDIR/data/7942gene.faa
|   |   |-- 8aa4a9d2fc.faa -> $WORKDIR/data/6803genes.faa
|   |   `-- a810086108.faa -> $WORKDIR/data/7942genes.faa
|   `-- makeblastdb
|       `-- c3ae472116
|           |-- c3ae472116.pdb.phr
|           |-- c3ae472116.pdb.pin
|           `-- c3ae472116.pdb.psq
|-- exprs
|   |-- list
|   |   `-- 7d562183ce.faa.list -> ../../cache/lines/c3ae472116.txt
|   |-- load_faa
|   |   |-- 33c746245c.faa -> ../../cache/load/82645ed6f6.faa
|   |   |-- 387f499ddc.faa -> ../../cache/load/a810086108.faa
|   |   `-- dee24d6c3c.faa -> ../../cache/load/8aa4a9d2fc.faa
|   |-- makeblastdb_prot
|   |   `-- a76d25f4b5.pdb -> ../../cache/lines/3f62a639a8.txt
|   |-- num
|   |   `-- 3e281fad0f.num -> ../../cache/lines/0cf27272d3.txt
|   |-- psiblast_train_db
|   |   |-- 5fbeb4e044_4d0e43d616_0aa7a0990a.pssm -> ../../cache/lines/4d395ef46f.txt
|   |   `-- 5fbeb4e044_4d0e43d616_0aa7a0990a.pssm.tmp
|   `-- str
|       |-- 1d8a61c770.str -> ../../cache/lines/bdc783547b.txt
|       |-- b9eaf59b0c.str -> ../../cache/lines/8abd22b6c2.txt
|       `-- ec3b181d8a.str -> ../../cache/lines/adfdb6bd8c.txt
`-- vars
    |-- pcc6803.faa -> ../exprs/load_faa/dee24d6c3c.faa
    |-- pcc7942.faa -> ../exprs/load_faa/387f499ddc.faa
    |-- pssm.pssm -> ../exprs/psiblast_train_db/5fbeb4e044_4d0e43d616_0aa7a0990a.pssm
    |-- query.faa -> ../exprs/load_faa/33c746245c.faa
    `-- result -> ../vars/pssm.pssm

13 directories, 29 files
