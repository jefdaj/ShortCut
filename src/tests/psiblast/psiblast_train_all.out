>Synpcc7942_0002 (trained on c3ae472116.pdb.list)
Last position-specific scoring matrix computed, weighted observed percentages...
            A   R   N   D   C   Q   E   G   H   I   L   K   M   F   P   S   T...
    1 L    -1  -2  -3  -4  -1  -2  -3  -4  -3   2   4  -2   2   0  -3  -2  -1...
    2 A     4  -1  -2  -2   0  -1  -1   0  -2  -1  -1  -1  -1  -2  -1   1   0...
...

